subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -1,179 +1,194 @@
 # SPDX-License-Identifier: Apache-2.0
 
+if(CONFIG_BT_CTLR_ADVANCED_FEATURES)
```

Comment:
is it possible to make the refactoring a separate commit from the move of the nrf sync ticker?